tools/js-tools/git-hooks/pre-push-hook.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+const { spawnSync } = require( 'child_process' );
+const chalk = require( 'chalk' );
+
+console.log( chalk.green( 'Checking if changelog files are needed. Just a sec...' ) );
+
+/**
+ * Checks if changelog files are required.
+ */
+function checkChangelogFiles() {
```

Comment:
The initial log should be inside the function too.
```suggestion
/**
 * Checks if changelog files are required.
 */
function checkChangelogFiles() {
	console.log( chalk.green( 'Checking if changelog files are needed. Just a sec...' ) );
```